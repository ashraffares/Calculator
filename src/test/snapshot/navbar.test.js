import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../../component/navbar';

describe('Navbar componenet', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
