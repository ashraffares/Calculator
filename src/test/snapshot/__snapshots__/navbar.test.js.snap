// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar componenet matches the snapshot 1`] = `
<nav
  className="nav"
>
  <h1>
    Math Magicians
  </h1>
  <div
    className="links"
  >
    <a
      exact={true}
      href="/"
      onClick={[Function]}
    >
      Home 
    </a>
    <a
      className="middle_link"
      href="/quate"
      onClick={[Function]}
    >
      Quate 
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator 
    </a>
  </div>
</nav>
`;
